sdk/communication/azure-communication-networktraversal/CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # Release History
 
-## 1.1.0b2 (Unreleased)
+## 1.1.0b2 (2024-04-03)
```

Comment:
Are we actually releasing this? If so, why?

Reply:
If we do need to release it - it should be v1.1.0, and the package version will need to be updated.

Reply:
Add a new version 1.1.1

Reply:
keep using 1.1.0b2 since this package never GA'ed